seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -1535,7 +1535,7 @@
         "cannot change the value of <code>myStr</code> to \"Job\", because the contents of <code>myStr</code> cannot be altered. Note that this does <em>not</em> mean that <code>myStr</code> cannot be changed, just that the individual characters of a <dfn>string literal</dfn> cannot be changed. The only way to change <code>myStr</code> would be to assign it with a new string, like this:",
         "<blockquote>var myStr = \"Bob\";<br>myStr = \"Job\";</blockquote>",
         "<h4>Instructions</h4>",
-        "Correct the assignment to <code>myStr</code> to achieve the desired effect."
+        "Correct the assignment to <code>myStr</code> to achieve the desired effect that is intended by <code>myStr[0] = "H";</code>."
```

Comment:
One more thing. The double quotes in the `"H"` need to be escaped, otherwise, the JSON file won't be parsed correctly.

```
...intended by <code>myStr[0] = \"H\";</code>."
```